app/assets/javascripts/initializers/initializeSwipeGestures.js

```diff
@@ -1,61 +1,93 @@
-function initializeSwipeGestures(){
-  initializeSwipeGestures.called = true;
-  swipeState = "middle";
-  setTimeout(function(){
+'use strict';
```

Comment:
this file unfortunately has cryptic variable names and no comments :( (outside of the scope of this PR)